force-app/main/default/triggers/CommunalUserTrigger.trigger

```diff
@@ -5,14 +5,10 @@ trigger CommunalUserTrigger on Communal_Service_User__c(
 ) {
   if (Trigger.isBefore) {
     if (Trigger.isDelete) {
-      CommunalServiceUserTriggerHandler.makeRegionsPrivate(Trigger.old);
       // CommunalServiceUserTriggerHandler.makeUserPrivate(Trigger.old);
```

Comment:
is this meant to be commented out?

Reply:
I haven`t yet completed a task with letting the user to read and edit own communal_user  account. There are 2 ways to achieve this: through apex sharing or changing communal_user `s owner. Since creation of a communal_user in DB should be done  after registration(which isn`t implemented yet), I imitated it by changing user`s owner. But if I cannot achieve letting the user to read and edit own communal_user  account through owner, I will use apex sharing instead, that is commented now.